data/brands/office/company.json

```
@@ -4,6 +4,16 @@
     "exclude": {"generic": ["^company$"]}
   },
   "items": [
+        {
+      "displayName": "Thales",
+      "locationSet": {"include": ["fr"]},
```

```suggestion
      "locationSet": {"include": ["001"]},
```
Thales is headquartered in France, but has offices in a number of countries: https://en.wikipedia.org/wiki/Thales_Group#Thales_international. Some of those have Wikidata items for the local organization (https://www.wikidata.org/wiki/Q1161666#P355 ), but I think it's fine to tag everything with the top-level group.